x-pack/plugins/security_solution/public/resolver/models/indexed_process_tree/isometric_taxi_layout.ts

```diff
@@ -501,13 +506,26 @@ const distanceBetweenNodesInUnits = 2;
  */
 const distanceBetweenNodes = distanceBetweenNodesInUnits * unit;
 
-export function nodePosition(
+/**
+ * @deprecated use `nodePosition`
+ */
+export function processPosition(
   model: IsometricTaxiLayout,
   node: SafeResolverEvent
 ): Vector2 | undefined {
   return model.processNodePositions.get(node);
 }
 
+export function nodePosition(model: IsometricTaxiLayout, nodeID: string): Vector2 | undefined {
+  // Find the indexed object matching the nodeID
+  // NB: this is O(n) now, but we will be indexing the nodeIDs in the future.
+  for (const candidate of model.processNodePositions.keys()) {
```

Comment:
:grey_question: 
`````js
const candidateID = model.processNodePositions.keys().find((v)=>{ return eventModel.entityIDSafeVersion(v) === nodeID });
return candidate && processPosition(model, candidateID);
`````

Reply:
Can we use `.find` on iterators?

Reply:
stage 2 proposal https://github.com/tc39/proposal-iterator-helpers

Reply:
Spread it, I guess? I don't have a real strong feeling on this one, feel free to skip this I was just nudging a way around the loops.

Reply:
I love loops